common/components/UI/RealEstateComponents/RealEstateModal/RealEstateForm/index.tsx

```diff
@@ -9,19 +9,22 @@ import { IExtendedRealestate } from '@common/api/realestateApi/realestate.api.ty
 
 interface Props {
   form: FormInstance<any>
-  currentRealEstate: IExtendedRealestate
+  currentRealEstate?: IExtendedRealestate
+  editable?: boolean
+  disabled?: boolean
 }
 
-const RealEstateForm: FC<Props> = ({ form, currentRealEstate }) => {
-  const initialValues = useInitialValues(currentRealEstate)
+const RealEstateForm: FC<Props> = ({
+  form,
+  currentRealEstate,
+  editable = true,
+  disabled = false,
+}) => {
+  const companyNameValue = Form.useWatch('companyName', form)
+  const descriptionValue = Form.useWatch('description', form)
 
   return (
-    <Form
-      form={form}
-      layout="vertical"
-      className={s.Form}
-      initialValues={initialValues}
-    >
+    <Form form={form} layout="vertical" className={s.Form}>
       {currentRealEstate ? (
         <Form.Item name="domain" label="Надавач послуг">
```

Comment:
Something happened to DomainSelect placeholder. Check this in UI, please
![image](https://github.com/user-attachments/assets/d09e6549-134a-4ab6-8c8a-a65d637cceaf)